conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -20,14 +20,24 @@ environment:
   # overridden in the matrix definition where appropriate.
   CONDA_PY: "27"
 
-  BINSTAR_TOKEN:
-    # Generated with "binstar auth --create --name appveyor -o conda-forge" and encrypted on ci.appveyor.com/tools/encrypt.
-    secure: wtylLGtp/ID3G3DFT2uxuLFXuijFqmxI6phj60izpNC3pxd585bR6k2oonf5iGGb
```

Comment:
This looks like a bug: at least it looks like the appveyor binstart tolen was never changed by the tooling (the call was also missing from the register-ci call